tools/wpt/utils.py

```diff
@@ -11,6 +11,13 @@
 
 class Kwargs(dict):
     def set_if_none(self, name, value, err_fn=None, desc=None, extra_cond=None):
+        """
+        :type name: str
```

Comment:
This is just documentation, right? Or is some tooling reading it? It doesn't look much like Python 3.5's type hints afaict, but I haven't used those so maybe I'm wrong.

Reply:
I've been using Pycharm and was just going off their autogenerated docstring -
 https://www.jetbrains.com/help/pycharm/type-hinting-in-pycharm.html . Will tweak settings to use PEP 484 instead and update this PR.